python/port/mod/time/modtime.cpp

```diff
@@ -1,17 +1,21 @@
 extern "C" {
 #include "modtime.h"
+#include <py/runtime.h>
 }
 #include <ion/timing.h>
 #include "../../helpers.h"
 #include <py/smallint.h>
-#include <py/runtime.h>
```

Comment:
We may want to either move all the `py/*.h` to the `extern "C"` block, or just leave them as is. I'd vode for the latter 😄

Reply:
I need the raiseValueError, which is C linked, so I'll move every `py/*.h` in the `extern "C"`